.github/workflows/release.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
         platforms: linux/amd64,linux/arm64
         build-args: |
           VERSION=${{ github.ref_name }}
+          GIT_COMMIT=${{ github.sha }}
+          GIT_TREE_STATE=clean
```

Comment:
The tree is always clean at this point in the workflow. Saying so explicitly is easier than adding additional steps to invoke a git command and assess this dynamically when we know what the result is.